app/views/review_queues/reviews.html.erb

```diff
@@ -0,0 +1,27 @@
+<h1>Review history for <%= @queue.name.titleize %> </h1>
+
+<% if @all %>
+  <%= link_to 'mine', all: 0 %> | <b>all</b>
+<% else %>
+  <b>mine</b> | <%= link_to 'all', all: 1 %>
+<% end %>
+
+<table class="table">
+  <tr>
+    <% if @all %> <th>User</th> <% end %>
+    <th>Task</th>
+    <th>Result</th>
+  </tr>
+
+    <% @reviews.each do |review| %>
+      <tr>
+        <% if @all %> <td><%= review.user.username %></td> <% end %>
+        <td><%= link_to review.item.reviewable.review_item_name, controller: :review_queues, action: :item, item_id: review.item.id %></td>
```

Comment:
`review.item.reviewable` is the same as `review`; good to have a named route again

Reply:
`review.item.reviewable` is the `SpamDomain` or `Post`, while `review` is the `ReviewResult`, isn't it?